src/backend_skia_bindings/image.rs

```diff
@@ -62,7 +64,7 @@ fn image_to_surface(image: &Image, surface: &mut [u8]) {
     use rgb::FromSlice;
 
     let mut i = 0;
-    if skia::ColorType::n32() == skia::ColorType::RGBA8888 {
+    if skia::ColorType::n32() == skia::ColorType::BGRA8888 {
```

Comment:
same as this - https://github.com/RazrFalcon/resvg/blob/b70a71293d86e4bd058970600668e4c9217810b5/resvg-skia/skia-bindings/skia_capi.cpp#L170